client/src/main/java/com/alibaba/nacos/client/utils/TenantUtil.java

```diff
@@ -27,6 +27,10 @@
     
     private static final String USER_TENANT;
     
+    private static final String ACM_NAMESPACE_PROPERTY = "acm.namespace";
```

Comment:
I remember `SystemPropertyKeyConst.ANS_NAMESPACE` is same.

Reply:
I will check it and resubmit.

Reply:
@KomachiSion I have changed to SystemPropertyKeyConst.ANS_NAMESPACE. the ACM_NAMESAPCE is kept as now. because it doesn't exist in SystemPropertyKeyConst.